int8 object_id       # object id number
int16 top_left_x       # X pixel value of top-left corner of bounding box
int16 top_left_y       # Y pixel value of top-left corner of bounding box
int16 bot_right_x      # X pixel value of bottom-right corner of bounding box
int16 bot_right_y 		# Y pixel value of bottom-right corner of bounding box